site/resources.html

```diff
@@ -0,0 +1,54 @@
+---
+layout: page
+title: Resources
+description: Sonobuoy Resources
+id: resources
+---
+
+<div class="sub-page">
+    <h2>Here you will find external resources about Sonobuoy, such as videos, podcasts, and community articles.</h2>
```

Comment:
@amymanion shouldn't this be h1?

Reply:
@ccbayer No, there is a "Resources" header at the top of the page which is the h1.